microcosm_flask/conventions/crud.py

```diff
@@ -209,6 +204,7 @@ def configure_crud(graph, obj, mappings, path_prefix=""):
         }
 
     """
+    ns = Namespace.make(ns, path=path_prefix)
```

Comment:
The older `configure_crud` invocations would take strings, classes, etc. We now expect a `Namespace`, but still support the older calls for compatibility via some logic in `Namespace.make()`